tm2/pkg/store/cache/store.go

```diff
@@ -172,6 +172,7 @@ func (store *cacheStore) dirtyItems(start, end []byte) {
 		return bytes.Compare(unsorted[i].Key, unsorted[j].Key) < 0
 	})
 
+	// #nosec G602
 	for e := store.sortedCache.Front(); e != nil && len(unsorted) != 0; {
```

Comment:
should we refactor this code and fix it?

Reply:
IMO, it's a false positive, protected by the `len(unsorted) != 0` we can see in line 176.

Looking for additional feedback, anyone?